demo/kserve/custom-manifests/caikit/caikit-tgis-isvc.yaml

```diff
@@ -5,14 +5,16 @@ metadata:
     serving.knative.openshift.io/enablePassthrough: "true"
     sidecar.istio.io/inject: "true"
     sidecar.istio.io/rewriteAppHTTPProbers: "true"
-  name: caikit-tgis-example-isvc
+  name: caikit-tgis-isvc
 spec:
   predictor:
+    serviceAccountName: sa
```

Comment:
serviceAccountName should not be added, won't be stable

Rest of the file looks fine

Reply:
Can you please explain what you mean? The SA is explicitly created in the **original** scenario and associated with the `storage-config` secret, but is never used, which is why the ISVC is not created successfully. We simply connected the dots.

Reply:
Yes, but the SA name will be added at creation time, and shouldn't be hardcoded. If the name doesn't match on the user's cluster (and there's a reasonable chance it won't), it'll error. I'll merge once this is fixed

Reply:
@Xaenalt : IIUC, it is ok with you that we use SA-based secret association, as long as we don't hard-code the SA name. If so, let me propose the following solution. In the step-by-step document, we already provide a **generic** yaml for the ISVC that provides a blank storage URI, and a **concrete** yaml that uses the Minio deployed earlier in the scenario. In accordance with your requirement above, the generic yaml will also contain a line for SA but **without specifying a name for SA**, only text like "add SA here that should have the secret for accessing the model". Similarly, in the concrete yaml, we will use the SA created in the example scenario. That way, if the user follows all the concrete steps, she/he ends up with a working ISVC that can be used for inference. Is this acceptable?

Reply:
@Xaenalt  Could I implement Erez' suggestion ?

Reply:
@Xaenalt  I pushed yesterday new template for ISVC and improved deploy-remove.md documentation file.  This should reflect Erez' proposal